src/common/transformations/src/transformations/op_conversions/convert_gather_to_compressed.cpp

```diff
@@ -89,8 +89,8 @@ ov::pass::ConvertGatherToGatherCompressed::ConvertGatherToGatherCompressed() {
         std::shared_ptr<ov::Node> gather_input_a =
```

Comment:
can we also update this variable?
std::shared_ptr<ov::Node> -> Output<Node>
reshape_const_to_2d(pattern_map.at(dicts_m).get_node_shared_ptr()) -> reshape_const_to_2d(pattern_map.at(dicts_m))